docs/articles/retrieval_augmented_generation_eval_qdrant_arize.md

```diff
@@ -0,0 +1,526 @@
+
+Please check out Phoenix [github]([https://github.com/Arize-ai/phoenix](https://github.com/Arize-ai/phoenix)) or [website]([https://phoenix.arize.com/](https://phoenix.arize.com/)) for more information.
+
+Let us talk about the RAG evaluation aspect of Phoenix in this article.
+
+Just like our [previous article]([https://superlinked.com/vectorhub/articles/retrieval-augmented-generation-eval-qdrant-ragas](https://superlinked.com/vectorhub/articles/retrieval-augmented-generation-eval-qdrant-ragas)) , we’ll be building a RAG pipeline and evaluate it using Phoenix.
+
+The code is available in the [github repo]([https://github.com/qdrant/qdrant-rag-eval/tree/master/workshop-rag-eval-qdrant-arize](https://github.com/qdrant/qdrant-rag-eval/tree/master/workshop-rag-eval-qdrant-arize)) if you prefer to follow along.
+
+There is also a walkthrough workshop [video]([https://www.youtube.com/watch?v=m_J0nFmnrPI](https://www.youtube.com/watch?v=m_J0nFmnrPI)) available for the reference.
+
+This Article will take you through :
+
+- Learning about key concepts and metrics in Phoenix
+- Evaluating a Naive RAG using Phoenix
+- Key observations
+- Building a [Hybrid RAG]([https://superlinked.com/vectorhub/articles/optimizing-rag-with-hybrid-search-reranking](https://superlinked.com/vectorhub/articles/optimizing-rag-with-hybrid-search-reranking))
+- Evaluating a Hybrid RAG using Phoenix
+- Concluding thoughts
+
+Lets begin!
+
+**Learning about key concepts and metrics in Phoenix**
+
+Phoenix is an open-source observability library designed for experimentation, evaluation, and troubleshooting. It allows AI Engineers to quickly visualize their data, evaluate performance, track down issues and optimize.
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_products.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_products.png)
+
+It provides a platform to enable evaluation and observability across different layers and stages of an LLM-based system.
+
+Arize offers a comprehensive suite of solutions to meet various needs, from open-source to enterprise-grade offerings. This [LLM tracing](https://docs.arize.com/phoenix/tracing/llm-traces-1) application provides visibility into the topology of your application whether you're running it locally in Jupyter notebook or a Docker container, or pushing your application into staging and production, the solutions provide unparalleled observability into every aspect of your LLM system.
+
+This includes tracing, prompt iteration, search and retrieval, [evaluating specific tasks](https://arize.com/blog-course/llm-evaluation-the-definitive-guide/), and ultimately improving your LLM through fine-tuning.
```

Comment:
I think we can cut this line

Reply:
Line 42

Reply:
I believe this makes sense to keep for further reference.